askama/src/filters/alloc.rs

```diff
@@ -118,14 +120,34 @@ pub fn format() {}
 /// # }
 /// ```
 #[inline]
-pub fn linebreaks(s: impl fmt::Display) -> Result<HtmlSafeOutput<String>, fmt::Error> {
-    fn linebreaks(s: &str) -> String {
-        let linebroken = s.replace("\n\n", "</p><p>").replace('\n', "<br/>");
-        alloc::format!("<p>{linebroken}</p>")
+pub fn linebreaks<S: fmt::Display>(source: S) -> Result<HtmlSafeOutput<Linebreaks<S>>, Infallible> {
+    Ok(HtmlSafeOutput(Linebreaks(source)))
+}
+
+pub struct Linebreaks<S>(S);
```

Comment:
Just to be sure: this commit is just about optimizing the rendering of the filters modified here, right?

Reply:
Sorry, I don't understand the question.